app/controllers/api/v1/rents_controller.rb

```diff
@@ -0,0 +1,26 @@
+module Api
+  module V1
+    class RentsController < ApplicationController
+      before_action :set_user
+
```

Comment:
you aren't verifying if the user is logged in, this is do it with devise gem, check this link https://github.com/plataformatec/devise#controller-filters-and-helpers

Reply:
Done